pallets/rc-migrator/src/lib.rs

```diff
@@ -646,13 +685,26 @@ pub mod pallet {
 							},
 						}
 					},
-				MigrationStage::WaitingForAh => {
+				MigrationStage::WaitingForAh { .. } => {
 					// waiting AH to send a message and to start sending the data. This stage may be
 					// skipped if AH is fast.
```

Comment:
In the past there was a case that this stage can be skipped, but I think it cannot be skipped any more. So maybe adjust the comment.

Reply:
updated